### ISSUE TITLE
Support database creation

### ISSUE THREAD

**Issue by sjoukedv** [digitalocean/do-operator#30]
I'd like to also create databases similar to `DatabaseUser` resources

**Comment by adamwg** [comment]
Making sure I understand what you want, since the naming is confusing:

We support creating and managing a DO Managed Database instance (which is called a Database Cluster in the DO docs and API) via the `DatabaseCluster` CRD. The API does also support creating a "database" within a database cluster; that isn't currently supported in the operator, but there's no reason it couldn't be.

Are you wishing to manage databases within a database cluster via the operator?

**Comment by sjoukedv** [comment]
> Making sure I understand what you want, since the naming is confusing:
> 
> We support creating and managing a DO Managed Database instance (which is called a Database Cluster in the DO docs and API) via the `DatabaseCluster` CRD. The API does also support creating a "database" within a database cluster; that isn't currently supported in the operator, but there's no reason it couldn't be.
> 
> Are you wishing to manage databases within a database cluster via the operator?

Yes, exactly I wish to create a Database from the do-operator which should be possible as the DO API also support this.

If you want to take it to the next level; there can be something like a `DatabasePermission` resource that grants privileges on the newly created database (with the do-operator) this allows developers to create everything dynamically from Kubernetes.

**Comment by adamwg** [comment]
Thanks for confirming! Agreed, this would be a good feature.
